opencensus-migration/src/main/java/SpanCache.java

```diff
@@ -0,0 +1,45 @@
+import io.opencensus.implcore.trace.RecordEventsSpanImpl;
+import io.opencensus.trace.Span;
+import java.util.HashMap;
+import java.util.Map;
+
+public class SpanCache {
+  private static SpanCache spanCache;
+  private Map<Span, io.opentelemetry.trace.Span> ocToOt = new HashMap<>();
```

Comment:
We may consider using thread-safe maps here, e.g. `ConcurrentHashMap`, and use `putIfAbsent` instead of standard `put`. Applies to both maps.